include/deal.II/base/symmetric_tensor.h

```diff
@@ -181,49 +181,6 @@ determinant(const SymmetricTensor<2, dim, Number> &);
 
 namespace internal
 {
-  // Workaround: The following 4 overloads are necessary to be able to
```

Comment:
These are not necessary anymore I presume?

Reply:
While we don't support those compilers any more these overloads are necessary for nvcc - I'm going to leave them back in for now.